src/components/Pages/Create/CreateProject.js

```diff
@@ -0,0 +1,183 @@
+import React, { useEffect, useState } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+
+import Header from '../../Header/Header'
+import TagCluster from '../../TagCluster/TagCluster'
+
+export default function Create({ onBack }) {
+  const [tags, setTags] = useState(null)
+  const [image, setImage] = useState()
+  const [newProject, setNewProject] = useState({})
+
+  useEffect(() => {
+    fetch('https://unfinished-api.herokuapp.com/api/tags')
+      .then((res) => res.json())
+      .catch((error) => console.log(error))
+      .then((tags) => setTags(tags))
+  }, [])
+  return (
+    <>
+      <Header title='Initiate a Project' onBack={onBack} />
+      <Form>
+        <label htmlFor='image'>
+          Select an image:
+          <img
+            alt='User uploaded'
+            src={image ? URL.createObjectURL(image) : '/images/image-missing.jpg'}
+          />
+        </label>
+        <input
+          type='file'
+          id='image'
+          accept='image/*'
+          onChange={(event) => setImage(event.target.files[0])}
+        />
+        <label htmlFor='location'>Where are you based?</label>
+        <input
+          id='location'
+          type='text'
+          pattern='.*\S.*'
```

Comment:
Why this pattern?

Reply:
It is valid if at least one non-whitespace character is entered. I use this as a visual indicator for the user whether an input is valid and this one becomes valid quickly and therefore reassures the user that he/she is on a good way. Any other recommendations?

Reply:
Ah, I see. It's good as is, keep it.